posts/2021-the-bottom-line.md

```diff
@@ -0,0 +1,177 @@
+---
+author: 'Mathlib community'
+category: 'overview'
+date: 2021-12-30 23:59:59+00:00
+description: ''
+has_math: true
+link: ''
+slug: 2021-the-bottom-line
+tags: ''
+title: 2021 - The Bottom Line
+type: text
+---
+
+Since the mathlib repository was created in summer 2017,
+each year has been bigger than the last. 
+As an end-of-year retrospective, we look at how the mathlib library and community have developed in 2021.
+
+# Statistics
```

Comment:
I think this should be heading level 2, since the main title is heading level 1

Reply:
This is intentional: https://github.com/leanprover-community/blog/pull/28

Reply:
Huh, I'm as surprised as you were. Thanks!